contracts/near-campaign/src/add_keys.rs

```diff
@@ -1,5 +1,8 @@
 use crate::*;
 
+// Access key allowance
+const ACCESS_KEY_ALLOWANCE: u128 = 200_000_000_000_000_000_000_000; // 0.2 NEAR
```

Comment:
0.2 NEAR will allow a user to call a method ~6 times. 0.2 / 0.03 (300Tgas price in NEAR). Maybe we can reduce it to 0.1 NEAR?